src/main/webapp/components/directives/cache/security/security.js

```diff
@@ -0,0 +1,158 @@
+(function () {
+  'use strict';
+
+  var module = angular.module('ispn.directives.cache.security', ['ispn.services.utils']);
+
+  module.directive('security', ['utils', '$modal', function (utils, modal) {
+    return {
+      restrict: 'E',
+      scope: {
+        data: '=',
+        cluster: '=',
+        metadata: '=',
+        initDefaults: '=',
+        readOnly: '=',
```

Comment:
Minor, but we don't actually use `initDefaults` or `readOnly` at present. Maybe we should utilise readOnly for the checkboxes ng-disabled field? Currently I don't think initDefaults is applicable as we only have checkbox inputs in this directive.